src/Http/Controllers/AppointmentController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace LibreEHR\FHIR\Http\Controllers;
+
+use Illuminate\Http\Request;
+use LibreEHR\FHIR\Adapters\FHIRAppointmentAdapter;
+use Illuminate\Support\Facades\DB;
+use phpDocumentor\Reflection\DocBlock\Tags\Return_;
+
+class AppointmentController extends AbstractController
+{
+    public function __construct()
+    {
+        $this->adapter = new FHIRAppointmentAdapter;
```

Comment:
This is not needed, use the IOC container bindings in Utilities/Providers/FHIRServiceProvider. In general, it is better practice to pass dependencies as parameters. Thanks.